platform/pulpcore/app/models/publication.py

```diff
@@ -0,0 +1,103 @@
+from django.db import models
+from django.core.files.storage import DefaultStorage
+
+from pulpcore.app.models import Model
+
+
+class Publication(Model):
+    """
+    Fields:
+        created (models.DatetimeField): When the publication was created UTC.
+
+    Relations:
+        publisher (models.ForeignKey): The publisher that created the publication.
+    """
+
+    created = models.DateTimeField(blank=True, null=True, auto_now_add=True)
```

Comment:
With `auto_now_add=True` this field should never be blank. Shouldn't `blank=True, null=True` be removed?